cfgov/jinja2/v1/the-bureau/leadership-calendar/index.html

```diff
@@ -86,8 +86,9 @@
                 'show_current_filters': false,
                 'action'              : 'pdf/',
                 'submit_label'        : 'Download PDF',
-                'additional_classes'  : 'js-validate_form-not-empty',
-                'form_method'         : 'post'
+                'additional_classes'  : 'js-validate_form-not-empty, js-validate_require-date',
```

Comment:
I'm going to leave as-is for now but will switch them all over at the same time.